packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -32,6 +34,20 @@ export interface Options {
    * Cause the bootstrap peer tag to be removed after this number of ms (default: 2 minutes)
    */
   tagTTL?: number;
+  /**
+   * The initial timeout before querying a peer for the first time (default: 5 seconds)
+   */
+  initialQueryTimeout?: number;
```

Comment:
This does not seem necessary, we want to query as soon as we can. Why adding a 5s delay to find new peers?

Reply:
It seems that queries are only started upon emission of event `"change:protocols"` which seems correct.
Meaning that should not be a situation where a request is fire "immediately". It should only be fire once a peer is connected, based on a connection event.